### ISSUE TITLE
vreplication needs to filter out UNKNOWN queries?

### ISSUE THREAD

**Issue by aquarapid** [vitessio/vitess#5725]
Some types of DDL/admin statements in the replication stream are not currently filtered out, and will lead to vreplication halting with an error.

E.g. (with some fake data):

```
'SELECT * FROM _vt.vreplication\G'
*************************** 1. row ***************************
                   id: 1
             workflow: SplitClone
               source: keyspace:"ks1" shard:"0" filter:<rules:<match:"t1" > >
                  pos: MySQL56/ffffffff-ffff-ffff-ffff-ffffffffffff:1-100001
             stop_pos: NULL
              max_tps: 9223372036854775807
  max_replication_lag: 9223372036854775807
                 cell: NULL
         tablet_types: NULL
         time_updated: 1579201328
transaction_timestamp: 1579186267
                state: Running
              message: vttablet: rpc error: code = Unknown desc = stream error @ ffffffff-ffff-ffff-ffff-ffffffffffff:1-100001: unexpected statement type UNKNOWN in row-based replication: "GRANT USAGE ON *.* TO 'xtrabackup'@'localhost'"
              db_name: db1
```

The offending statement here is :  `GRANT USAGE ON *.* TO 'xtrabackup'@'localhost'`

We should filter other types of similar commands too, if we are not already, e.g. CREATE USER

**Comment by aquarapid** [comment]
This will happen for `REVOKE`, in addition  to `GRANT` too, of course.

**Comment by rohit-nayak-ps** [comment]
Fixed via #5775
